1.10/networking/load-balancing-vips/index.md

```diff
@@ -1,134 +1,50 @@
 ---
-post_title: Load Balancing and VIPs
-nav_title: Load Balancing and VIPs
+post_title: Load Balancing and Virtual IPs (VIPs)
 menu_order: 00
 ---
 
-DC/OS comes with an east-west load balancer that's meant to be used to enable multi-tier microservices architectures. It acts as a TCP Layer 4 load balancer, and it's tightly integrated with the kernel. 
+DC/OS provides an east-west load balancer (minuteman) that enables multi-tier microservices architectures. It acts as a TCP layer 4 load balancer, and it's tightly integrated with the Mesos kernel. The features include:
 
-## Usage
-You can use the layer 4 load balancer by assigning a [VIP from the DC/OS web interface](/docs/1.10/networking/load-balancing-vips/virtual-ip-addresses/). Alternatively, if you're using something other than Marathon, you can create a label on the [port](https://github.com/apache/mesos/blob/b18f5bf48fda12bce9c2ac8e762a08f537ffb41d/include/mesos/mesos.proto#L1813) protocol buffer while launching a task on Mesos. This label's key must be in the format `VIP_$IDX`, where `$IDX` is replaced by a number, starting from 0. Once you create a task, or a set of tasks with a VIP, they will automatically become available to all nodes in the cluster, including the masters.
+- Included with DC/OS by default 
+- Distributed load balancing of applications
+- Facilitates east-west communication within the cluster
+- User specifies a virtual IP in Marathon application
```

Comment:
User generally does not specify a virtual IP. He generally specifies and FQDN to address the service.